examples/experimental/i3s-picking/app.js

```diff
@@ -165,11 +168,40 @@ export default class App extends PureComponent {
         onTilesetLoad: this._onTilesetLoad.bind(this),
         onTileLoad: () => this._updateStatWidgets(),
         onTileUnload: () => this._updateStatWidgets(),
+        onClick: info => this.handleRenderAttributes(info),
+        pickable: true,
         loadOptions
       })
     ];
   }
 
+  async handleRenderAttributes(info) {
+    const {attributeStorageInfo} = this.state;
+    console.log('color', info.color); //eslint-disable-line
+
+    const attributeUrls = info.object.header.attributeUrls;
+    const attributes = [];
+
+    for (let index = 0; index < attributeStorageInfo.length; index++) {
+      const url = attributeUrls[index];
+      const attributeName = attributeStorageInfo[index].name;
+      const attributeType = this.getAttributeValueType(attributeStorageInfo[index]);
+      const attribute = await load(url, I3SAttributeLoader, {attributeName, attributeType});
```

Comment:
Can't we get these from the tile? Do we need to load them again here?

Reply:
Each tile has many features (buildings) We just take a list of attributes such as builiding Id, building name e.t.c. I guess when user clicked on particular building on the map we should get an objectId and select the rest attributes from current attributes.

Reply:
Tile object doesn't have this feature attributes inside.